rotkehlchen/chain/optimism/modules/compound/v3/decoder.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.compound.v3.decoder import Compoundv3CommonDecoder
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+from .constants import COMPOUND_BULKER_ADDRESS, COMPOUND_REWARDS_ADDRESS
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.base.node_inquirer import BaseInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
```

Comment:
this isn't needed

Reply:
huh weird thtat normal linters did not even say anything. Thanks